CoreScriptsRoot/LoadingScript.lua

```diff
@@ -456,16 +451,13 @@ if isTenFootInterface then
 	createTenfootCancelGui()
 end
 
-local removedLoadingScreen = false
 local setVerb = true
-local lastRenderTime = nil
+local lastRenderTime, lastDotUpdateTime, brickCountChange = nil
```

Comment:
this is technically ok (all variables in list will be nil), but should be set to equal nil, nil, nil